packages/govuk-frontend/package.json

```diff
@@ -16,12 +17,13 @@
       "import": "./dist/govuk-esm/all.mjs",
       "require": "./dist/govuk/all.js"
     },
-    "./*": "./*"
+    "./*": "./*",
+    "./dist/": "./dist/",
+    "./package.json": "./package.json"
```

Comment:
**question**: Is there a way we can allow "./" in a similar fashion as "./*"? That'd let us not worry about which folders are specifically accessible 😊

Reply:
Ah sorry, I started with that but Visual Studio Code gives me this:

>Property ./ is not allowed.

Are there any new paths in v5 we should add for legacy Node.js?

Reply:
This was my thinking process by the way:

1. Take the compatibility subpath exports from [v4.6.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.6.0):

```json
"./govuk/": "./govuk/",
"./govuk-esm/": "./govuk-esm/",
"./package.json": "./package.json"
```

2. Convert compatibility subpath exports for v5

```json
"./dist/govuk/": "./dist/govuk/",
"./dist/govuk-esm/": "./dist/govuk-esm/",
"./package.json": "./package.json"
```

3. Simplify compatibility subpath exports:

```json
"./dist/": "./dist/",
"./package.json": "./package.json"
```

Reply:
Mind if I just do `govuk-prototype-kit.config.json` as Node.js can't require/import Markdown?

Reply:
Done it, but let me know